### PR TITLE
Add cleanup jobs for perf tests

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
Add image cleanup jobs for perf tests

/cc @chaodaiG

### CODE REVIEW

File: config/prod/prow/jobs/custom-jobs.yaml [hunk 1]
```diff
@@ -129,6 +129,43 @@ periodics:
...
+  name: ci-knative-cleanup-performance-tests
+  labels:
+    prow.k8s.io/pubsub.project: knative-tests
+    prow.k8s.io/pubsub.topic: knative-monitoring
+    prow.k8s.io/pubsub.runID: ci-knative-cleanup-performance-tests
+  agent: kubernetes
+  decorate: true
+  cluster: "build-knative"
+  decoration_config:
+    timeout: 21600000000000
```
Comment (chaodaiG) [inline config/prod/prow/jobs/custom-jobs.yaml:142]: why not use human readable format?

**Comment by chaodaiG** [comment]
/lgtm

/hold
in case you want to resolve my comment

File: config/prod/prow/jobs/custom-jobs.yaml (hunk 1 above)
Comment (chizhg) [reply config/prod/prow/jobs/custom-jobs.yaml]: Changed to `6h`, thanks.

**Pull request merged**